docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+Stable Patch Format
```

Comment:
https://docs.ubuntu.com/styleguide/en/#headings

We use sentence case unless it's a proper noun.
So "Stable patch format" in this case.

Thanks!

Reply:
Done. Thanks!